php/Terminus/Models/Environment.php

```diff
@@ -648,9 +648,23 @@ public function sendCommandViaSsh($command)
             'ssh -T %s@%s -p %s -o "AddressFamily inet" %s',
             [$sftp['username'], $sftp['host'], $sftp['port'], escapeshellarg($command),]
         );
+
+        // Catch Terminus running in test mode
+        $test_mode = isset($_ENV["TERMINUS_TEST_MODE"])
+            && filter_var($_ENV["TERMINUS_TEST_MODE"], FILTER_VALIDATE_BOOLEAN);
+        if ($test_mode) {
```

Comment:
It looks like `TERMINUS_TEST_MODE` is [not available](https://travis-ci.org/pantheon-systems/terminus/jobs/163987824#L4200) in the behat context?
@TeslaDethray @bensheldon: Is that true? Any ideas here?